api/chronos/urls.py

```diff
@@ -6,7 +6,7 @@
 
 urlpatterns = [
     url(r'^journals/$', views.ChronosJournalList.as_view(), name=views.ChronosJournalList.view_name),
-    url(r'^journals/(?P<journal_id>\w+)/$', views.ChronosJournalDetail.as_view(), name=views.ChronosJournalDetail.view_name),
+    url(r'^journals/(?P<journal_id>[0-9]{4}-[0-9]{4})/$', views.ChronosJournalDetail.as_view(), name=views.ChronosJournalDetail.view_name),
```

Comment:
Discussed privately, please make this `[-0-9A-Za-z]+`, as not all work with the above regex